www/tutorial.md

```diff
@@ -470,7 +470,10 @@ currently four ways to configure `--match`:
 * `--match prefix`: Check if the given string is a prefix of a benchmark
   path. For instance, `"foo"` will match `"foobar"`.
 
-* `--match glob`: Use the given string as a Unix-style glob pattern.
+* `--match glob`: Use the given string as a Unix-style glob pattern. Bear in
+  mind that performing a glob match on benchmarks names is done as if they were
+  file paths, so for instance both `"*/ba*"` and "*/*" will match "foo/bar",
```

Comment:
Make sure to surround all of these strings in backticks (for consistency with the rest of the tutorial).

Reply:
Oops, I missed that. It's fixed now.